src/redux/actions/vendorActions.js

```diff
@@ -26,3 +26,17 @@ export const registerBusiness = ({businessname,description,residence,phone}) =>
     }
     );
 };
+// Get all services for a particular vendor
+export const getServices = (id)=>(dispatch,getState)=>{
+  const config = tokenConfig(getState);
+  axios.get(`${url}/services/vendors/${id}`,config)
```

Comment:
Directly use `tokenConfig(getState)`. No need to create a new variable

Reply:
done